linux_os/guide/services/sssd/sssd_offline_cred_expiration/oval/shared.xml

```diff
@@ -25,7 +25,7 @@
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="obj_sssd_offline_cred_expiration" version="1">
     <ind:filepath>/etc/sssd/sssd.conf</ind:filepath>
-    <ind:pattern operation="pattern match">^\[pam]([^\n]*\n+)+?offline_credentials_expiration[\s]+=[\s]+1$</ind:pattern>
+    <ind:pattern operation="pattern match">^\[pam](?:[^\n]*\n+)+?offline_credentials_expiration[\s]+=[\s]+1$</ind:pattern>
```

Comment:
@matusmarhefka why this change?

Reply:
@redhatrises So why would you want to capture this group? It just doesn't make sense to me and that is why I rather set it as a group without capture.

Reply:
It's not needed to do that, the rule works without that, but it's better.

Reply:
I agree with Matus - explicit is better.